python/python/ert_gui/simulation/simulation_panel.py

```diff
@@ -74,7 +74,7 @@ def addSimulationConfigPanel(self, panel):
         self._simulation_stack.addWidget(panel)
         simulation_model = panel.getSimulationModel()
         self._simulation_widgets[simulation_model] = panel
-        self._simulation_mode_combo.addItem(str(simulation_model), simulation_model)
+        self._simulation_mode_combo.addItem(simulation_model.__repr__(),simulation_model)
```

Comment:
Slightly awkward calling `__repr__()` in my opinion. I would prefer a simple static method for each class returning the name and call that instead.